.github/workflows/ci.yaml

```diff
@@ -0,0 +1,155 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: CI
+
+on:
+  push:
+    branches: [ $default-branch, add_github_actions ]
```

Comment:
Are you sure `$defualt-branch` works?

Reply:
As documented here: https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/sharing-workflows-with-your-organization#creating-a-workflow-template